### PR TITLE
Al powder

### PR DESCRIPTION
**Pull request by elena-pascal**
Al powder calibrant data for testing used on powder_calibrate dials branch

### CODE REVIEW

File: Al_standard/imported.expt [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+      "detector": 0,
+      "goniometer": 0,
+      "scan": 0,
+      "imageset": 0
+    }
+  ],
+  "imageset": [
+    {
+      "__id__": "ImageSequence",
+      "template": "/home/elena/Work/Diamond/Data/data-files/Al_standard/0p67_5s_####.mrc",
```
Comment (Anthchirp) [inline Al_standard/imported.expt:17]: reference to absolute path

Comment (dagewa) [reply Al_standard/imported.expt:17]: I think you just edit the path in the file to be relative and it will work fine

Comment (elena-pascal) [reply Al_standard/imported.expt:17]: It seems that if I call `dials.import template=file.mrc` it won't save the path in `imported.expt`

Comment (dagewa) [reply Al_standard/imported.expt:17]: Yeah I don't think there's any way other than getting down and dirty with a text editor

**Pull request merged**